src/main.rs

```diff
@@ -112,9 +112,7 @@ fn play_pattern(pin: &mut OutputPin, config: &json::Value, pattern: &str, repeat
 }
 
 fn setup_mqtt_client(config: &json::Value) -> mqtt::Client {
-    let mqtt_broker = config["mqtt"]["broker"]
```

Comment:
Also strange, I did run fmt a couple of times and I could've sworn that it actually decided to spread that statement over multiple lines. Probably I removed something and now it is acceptable on a single line again...